app/models/application_form.rb

```diff
@@ -154,6 +154,14 @@ def candidate_can_choose_single_course?
     apply_2?
   end
 
+  def must_be_carried_over?
+    if ended_without_success?
+      recruitment_cycle_year < RecruitmentCycle.current_year || EndOfCycleTimetable.between_cycles_apply_2?
+    elsif !submitted?
+      recruitment_cycle_year < RecruitmentCycle.current_year && !EndOfCycleTimetable.between_cycles_apply_1?
```

Comment:
This means that you can't now carry over an un-submitted application until the new cycle opens. I don't think that's right (but I'm not sure), it's certainly a change of behaviour.

Reply:
To the best of my knowledge that wasn't specified behaviour; it's better to be consistent and use the same checks in multiple places.

Reply:
Agreed consistency is most important. The one-off task that already 'failed' any un-submitted applications around 18 Sept should have moved all these into a state that can be carried over anyway (in the same way as other unsuccessful applications). So this piece of business logic will pick up un-submitted applications too.